compat/faux_mmap.c

```diff
@@ -0,0 +1,80 @@
+/**
+ * @file faux_mmap.c
+ * @brief ly_mmap() and ly_munmap()
+ *
+ * Copyright (c) 2015-2021 CESNET, z.s.p.o.
+ *
+ * This source code is licensed under BSD 3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://opensource.org/licenses/BSD-3-Clause
+ */
+
+#include "common.h"
```

Comment:
`compat` is meant to have only implementations of standard functions, this is *libyang* specific (the `common.h` include). Sorry for not mentioning any details, please either put these functions in a separate `common_win.c` or something like that to `libyang/src` (if you anticipate more functions may be added there later) or put these functions directly in `common.c`. I do not mind having `#ifdef WIN32` with a single function call in another Linux function, I am trying to avoid having large Windows-specific code block in a Linux function, that is all. So you may keep `ly_mmap` as it was and just call Windows code as needed with code `ly_win_mmap();`, for example.

Reply:
OK, moved back to `common.c`. The reason why I have not done it like that in my first series is because there's some overlap in the leading code of `ly_mmap` and I want to use that.